osp/osp.c

```diff
@@ -410,6 +419,13 @@ osp_get_vts_ext (osp_connection_t *connection, osp_get_vts_opts_t opts,
   if (vts == NULL)
     return 1;
 
+  if (opts.version_only == 1)
```

Comment:
What was the purpose of adding this?  Should osp_get_vts_version be calling osp_get_vts_ext?